game/plugin/skills/fishing/src/org/apollo/game/plugin/skills/fishing/FishingSpot.kt

```diff
@@ -77,14 +77,19 @@ enum class FishingSpot(val npc: Int, private val first: Option, private val seco
         /**
          * A [FishingSpot] [Option] that can provide a two different types of fish.
          */
-        private data class Pair(override val tool: FishingTool, val primary: Fish, val secondary: Fish) : Option() {
+        private class Pair(override val tool: FishingTool, val primary: Fish, val secondary: Fish, vararg val junk: Fish) : Option() {
```

Comment:
Why did you remove the `data` modifier?

Reply:
There's also no reason for `junk` to be variadic. This constructor isn't a user facing API.

Reply:
The `data` modified was removed as you can't have variadic values in a `data` class. I'll fix this though, thanks!